src/Validation/ScannableURLsRestController.php

```diff
@@ -149,7 +149,12 @@ public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAna
 			}
 		}
 
-		$urls = $this->scannable_url_provider->get_urls();
+		$urls = array_filter(
+			$this->scannable_url_provider->get_urls(),
+			static function ( $item ) {
+				return is_array( $item ) && isset( $item['url'] );
```

Comment:
I remain confused how the `url` key could ever be undefined, given that `\AmpProject\AmpWP\Validation\ScannableURLProvider::get_urls()` constructs the `$urls` array any provides the `url` every time.